src/main/app/case/definition.ts

```diff
@@ -148,6 +148,17 @@ export const enum ConfidentialAddress {
   SHARE = 'share',
   KEEP = 'keep',
 }
+
+export const enum Connection {
+  A = 'A',
+  B = 'B',
+  C = 'C',
+  D = 'D',
+  E = 'E',
+  F = 'F',
+  G = 'G',
```

Comment:
Perhaps it might be nice to have some sort of human readable connection description, even if we're only sending A-G to the API e.g.

```suggestion
  RESIDUAL_JURISDICTION = 'G',
```